rdmo/projects/assets/js/interview/utils/page.js

```diff
@@ -10,6 +10,8 @@ const checkQuestion = (question, set) => {
 
 const initQuestionSet = (questionset) => {
   questionset.elements.forEach((element) => {
+    element.parent = questionset
```

Comment:
This is technically not needed anymore, but maybe we need it in the future.

Reply:
Are you sure it is not a [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)?